services/order.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
```

Comment:
The `get` method will raise a `DoesNotExist` exception if no user is found with the given `username`. Consider handling this exception to prevent potential crashes.